### ISSUE TITLE
GrayScale data 

### ISSUE THREAD

**Issue by AboMad** [christiancosgrove/pytorch-spectral-normalization-gan#18]
Hi there,
I am trying to apply this SNGAN implementation on grayscale cell images, my data size is quite enough large ~ 100,000 images. I have used resnet architecture models edited by adding an extra layer to generate/discriminate 64 pxl images and gan loss. By training the model about 15K iterations (~ 10 epochs) I could not recognize visual improvement for the generated samples as they suffer from checkboard and grid artifacts. 
The training curves are shown below:

![Disc_losses](https://user-images.githubusercontent.com/11579065/90311269-131a3700-df34-11ea-8166-c8eaa077c13a.jpg)
![Gen_losses](https://user-images.githubusercontent.com/11579065/90311270-13b2cd80-df34-11ea-82ea-13a258e266a7.jpg)

 I am not sure if I have to train for a longer time (more epochs), however, the training curves and the visual samples indicate abnormal case!
Any advice, please.
